server/service/integration_mdm_test.go

```diff
@@ -719,6 +719,145 @@ func (s *integrationMDMTestSuite) TestAppleProfileManagement() {
 
 	s.checkMDMProfilesSummaries(t, nil, expectedNoTeamSummary, &expectedNoTeamSummary) // empty because host was transferred
 	s.checkMDMProfilesSummaries(t, &tm.ID, expectedTeamSummary, &expectedTeamSummary)  // host still verifying team profiles
+
+	// add a new profile to the team
+	mcUUID := "a" + uuid.NewString()
```

Comment:
not a deal breaker, but how painful would it be to move this to it's own test? I don't know if you have experienced this, but we tend to make these integration tests super large, and then we keep on piling up until they're super hard to parse and fix.

Reply:
Yeah, I was thinking about that but was torn between trying to get this done more quickly by reusing some of the setup in the existing tests. I'll try to refactor it when I do the activities.